src/Serilog/Log.cs

```diff
@@ -1201,5 +1209,16 @@ public static bool BindProperty(string propertyName, object value, bool destruct
         {
             return Logger.BindProperty(propertyName, value, destructureObjects, out property);
         }
+
+        /// <summary>
+        /// An <see cref="ILogger"/> instance that silently ignores all log messages
```

Comment:
Nitpick, could use a `.` here.

Perhaps we could try `"that efficiently ignores all method calls"`?

Reply:
👍 
Shall do